ckanext/justicehub_theme/templates/snippets/social.html

```diff
@@ -0,0 +1,16 @@
+{% set current_url = h.full_current_url() %}
+{% block social %}
+  <section class="module module-narrow social">
+    {% block social_title %}
+      <h2 class="module-heading social-heading"><i class="fa fa-share-square-o"></i> {{ _('Share on') }}</h2>
+    {% endblock %}
+    {% block social_nav %}
+    <div class="social-links">
+      <ul class="list-unstyled list-inline">
+        <li><a href="https://twitter.com/share?url={{ current_url }}" target="_blank"><i class="fa fa-twitter fa-lg"></i></a></li>
+        <li><a href="https://www.facebook.com/sharer.php?u={{ current_url }}" target="_blank"><i class="fa fa-facebook fa-lg"></i></a></li>
+      </ul>
+    </div>
+    {% endblock %}
+  </section>
+{% endblock %}
```

Comment:
the mobile view has some issues.

![image](https://user-images.githubusercontent.com/30413611/82761147-0d6df300-9e16-11ea-9ad3-4118dc5e1d5a.png)